City-of-Helsinki/drupal-hdbt
# UHF-2457: Style unit search language checkbox

# Style unit search language checkbox

[UHF-2457](https://helsinkisolutionoffice.atlassian.net/browse/UHF-2457)

## What was done

* Discussed with design Elina about code limitations on positioning the checkbox location
* Proposed a new design that was approved (design specs will be updated to match what is shown here)
* Added a simple style to move the checkbox to a new line with proper margins on desktop and mobile.

![image](https://user-images.githubusercontent.com/1191667/134639833-79fe8097-1bb5-44b8-9ea3-d96532ab2142.png)
![image](https://user-images.githubusercontent.com/1191667/134639873-78afa2b1-c952-44a6-ba0d-5a53c5c6709e.png)

## How to test

* Get the language checkbox enabled in unit search components following [backend instructions](https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/87).
* Notice that the checkbox is positioned as seen in above images both on desktop and mobile
* Verify that SCSS code is ok. Especially the targeting of the checkbox is a bit iffy, but that's where we ended up with Lauri.

Related: https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/87
